sdk/src/types/block/rand/output/mod.rs

```diff
@@ -43,7 +43,6 @@ pub fn rand_output_id() -> OutputId {
 
 /// Generates a random [`BasicOutput`](BasicOutput).
 pub fn rand_basic_output(token_supply: u64) -> BasicOutput {
-    // TODO: Add `NativeTokens`
```

Comment:
Because it's supposedly done in rand_allowed_features ?

Reply:
Yes, already done like this, read PR description :trollface:

Reply:
oopsie